api-reference/10 UI Components/Errors and Warnings/W1018.md

```diff
@@ -0,0 +1,20 @@
+---
+id: ErrorsUIWidgets.W1018
+---
+---
+##### shortDescription
+Occurs in the DataGrid when infinite scrolling is used and **selection**.[allowSelectAll](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#allowSelectAll) is set to **true**.
```

Comment:
I think we should avoid the gerund form ("infinite scrolling"). Do _users_ scroll? If so, can we say the following instead?

`Occurs in the DataGrid when users can scroll infinitely and **selection**.[allowSelectAll](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#allowSelectAll) is set to **true**.`

Reply:
The gerund form can be here because there's the [scrolling ](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/) prop.